doqclient.go

```diff
@@ -144,7 +143,7 @@ func (d *DoQClient) Resolve(q *dns.Msg, ci ClientInfo) (*dns.Msg, error) {
 
 	// Write the query into the stream and close is. Only one stream per query/response
 	_ = stream.SetWriteDeadline(time.Now().Add(time.Second))
-	if _, err = stream.Write(p); err != nil {
+	if _, err = stream.Write(b); err != nil {
```

Comment:
Whoops, that must be my bug. Thanks for the fix!

Reply:
np